### ISSUE TITLE
Impossible [SequelizeDatabaseError: ER_NO_SUCH_TABLE: Table 'db.table_name' doesn't exist]?

### ISSUE THREAD

**Issue by liuzhen2008** [sequelize/sequelize#6360]
I have a node script that connects to a mysql database using sequelize and do a bunch of read requests.

The script ran for like 2 hours doing varies stuff (mostly data conversion), and after about 2 hours, it decides to throw a table not exist error when querying something like "SELECT FROM blah blah JOIN table_name on blah blah WHERE id = 3".

First of all, the same query was ran about 1 second ago with a different id on where clause, and it ran every 30 seconds or so for the previous 2 hours. I manually checked the mysql database after I saw the script exits due to the error and know for a fact that that the table "table_name" exists.

The only functions I used in sequelize in my scirpt are new Sequelize("DB_URI") and sequelize.query()

So I am wondering is there some other reason that might trigger this error? (Is it possible that, for example, the connection to db was lost during the query, and the error is table not exist?)

The body of the error looks something like this. I replaced the actual table_name with 'table_name' and simplified the sql.

```
{ [SequelizeDatabaseError: ER_NO_SUCH_TABLE: Table 'db.table_name' doesn't exist]
  name: 'SequelizeDatabaseError',
  message: 'ER_NO_SUCH_TABLE: Table \'db.table_name\' doesn\'t exist',
  parent:
   { [Error: ER_NO_SUCH_TABLE: Table 'db.table_name' doesn't exist]
     code: 'ER_NO_SUCH_TABLE',
     errno: 1146,
     sqlState: '42S02',
     index: 0,
     sql: 'SELECT * FROM db.table_name' },
  original:
   { [Error: ER_NO_SUCH_TABLE: Table 'db.table_name' doesn't exist]
     code: 'ER_NO_SUCH_TABLE',
     errno: 1146,
     sqlState: '42S02',
     index: 0,
     sql: 'SELECT * FROM db.table_name' },
  sql: 'SELECT * FROM db.table_name' }

```

**Comment by janmeier** [comment]
Can you create an [SSCCE](HTTP://SSCCE.ORG/)? Its impossible for us to debug the problem if we can't replicate it

**Comment by liuzhen2008** [comment]
@janmeier I understand this is more of a "open ended" question that I am asking. I was trying to reproduce and narrow down the issue as well. Do you know if this error is some error that mysql throws and sequelize merely captured, or something that sequelize throws?

**Comment by janmeier** [comment]
This is definitely a mysql error - are you sure they queries look absolutely identical?

**Comment by liuzhen2008** [comment]
@janmeier  In that case I suppose there is nothing you guys can do.

I am pretty sure the query has been run like thousands of times before it break. I was hoping my ops guys decided to move the data around on a friday night but apparently they didn't.

Thanks for your help.

(I first encountered the issue on a local machine, and I just tested it again on a cloud server, it ran pass the problematic id without any issue..)
